codalab_service.py

```diff
@@ -667,7 +659,9 @@ def run_service_cmd(self, cmd, root=False, service='rest-server'):
 
     @staticmethod
     def wait(host, port, cmd):
-        return '/opt/wait-for-it.sh {}:{} -- {}'.format(host, port, cmd)
+        return '/opt/wait-for-it.sh --timeout={} {}:{} -- {}'.format(
```

Comment:
I noticed that creating a new temp instance can take up to 4 minutes. I added a timeout here.